operator/pkg/utils/testutils/client.go

```diff
@@ -3,6 +3,7 @@ package testutils
 import (
 	"testing"
 
+	osconfigv1 "github.com/openshift/api/config/v1"
```

Comment:
This can be removed right?

Reply:
It was used for the tests because there were real `openshift/api/config/v1` objects for the fake client. But I do not have a strong preference about using real object in the tests. I tuned tests to use `unstructured` so schema registration is redundant in this case.